common/lib/common-utils/src/hashFileBrowser.ts

```diff
@@ -17,8 +17,14 @@ import { IsoBuffer } from "./bufferBrowser";
 export async function hashFile(file: IsoBuffer): Promise<string> {
     // Handle insecure contexts (e.g. running with local services)
     // by deferring to Node version, which uses a hash polyfill
+    // When packed, this chunk will show as "FluidFramework-HashFallback" separately
+    // from the main chunk and will be of non-trivial size.  It will not be served
+    // under normal circumstances.
     if (crypto.subtle === undefined) {
-        return import("./hashFileNode").then(async (m) => m.hashFile(file));
+        return import(
+            /* webpackChunkName: "FluidFramework-HashFallback" */
```

Comment:
Should you use `[index]` or `[request]` somewhere? Or is this guaranteed to only ever be once?

Reply:
`[index]` is the default naming we were getting I think.  I didn't use `[request]` because I think seeing a chunk named `hashFileNode.js` might be just as confusing as to why it's there.  It should only ever be downloaded never/once (not entirely sure what you mean by this question)

Reply:
I was thinking you could inject [index] or [request] anywhere in the string like a template. Something like:
```ts
/* webpackChunkName: "FluidFramework-HashFallback-[index]" */
```
But if there's really just the one for `hashFileNode.js`, then it's probably totally fine.